src/skin/legacyskinparser.cpp

```diff
@@ -316,7 +319,19 @@ QWidget* LegacySkinParser::parseSkin(QString skinPath, QWidget* pParent) {
         bool ok = false;
         double value = QString::fromStdString(attribute.value()).toDouble(&ok);
         if (ok) {
-            ControlObject::set(configKey, value);
+            // Set the specified attribute, possibly creating the control
+            // object in the process.
+            bool created = false;
+            // Update the value specified in the skin with whatever is in the
+            // saved config, if any.
+            value = m_pConfig->getValueString(configKey,
+                                              QString::number(value)).toDouble();
+            ControlObject* pControl = controlFromConfigKey(configKey, true,
```

Comment:
We should allow to set a persist key here, else all new COs are persistent and you cannot set a skin default.